pkg/docker/registry.go

```diff
@@ -166,16 +214,266 @@ func (registry *RegistryClient) GetImageInfo(repository string, tag string) (*ru
 	}, nil
 }
 
-func (registry *RegistryClient) GetTags(repository string) (*TagsAPIResponse, error) {
-	url := fmt.Sprintf("%s/v2/%s/tags/list", registry.address, repository)
+//LayerExists checks if layer exists in registry
+func (registry *RegistryClient) LayerExists(ctx context.Context, repository string, layerDigest string) (bool, error) {
```

Comment:
Trenger man virkelig to returverdier her? Kan ikke return false, nil erstattes med en error?